local_pathfinding/coord_systems.py

```diff
@@ -65,3 +65,22 @@ def latlon_to_xy(reference: LatLon, latlon: LatLon) -> XY:
         x=distance * math.sin(true_bearing),
         y=distance * math.cos(true_bearing),
     )
+
+
+def xy_to_latlon(reference: LatLon, xy: XY) -> LatLon:
```

Comment:
@jamenkaye will be writing tests for this

Reply:
Added tests!